source/auth/tests/mongodb-aws.rst

```diff
@@ -60,6 +61,27 @@ Drivers MUST be able to authenticate using temporary credentials returned from a
 
   mongodb://<AccessKeyId>:<SecretAccessKey>@localhost/?authMechanism=MONGODB-AWS&authMechanismProperties=AWS_SESSION_TOKEN:<Token>
 
+Assume Role with Web Identity
+=============================
+
+Drivers MUST be able to authentiate using a valid OIDC token and associated
+role ARN taken from environment variables, respectively:
+
+.. code-block::
+  AWS_WEB_IDENTITY_TOKEN_FILE
+  AWS_ROLE_ARN
+  AWS_ROLE_SESSION_NAME (optional)
```

Comment:
Since `AWS_ROLE_SESSION_NAME` is optional, we should require drivers to test both with and without this environment variable set.

Reply:
Updated

Reply:
(Also updated https://github.com/mongodb/mongo-python-driver/pull/1032)